provisioning/service/src/ProvisioningServiceClient.cs

```diff
@@ -124,131 +124,17 @@ protected virtual void Dispose(bool disposing)
             }
         }
 
-        /// <summary>
-        /// Create or update a individual Device Enrollment record.
-        /// </summary>
-        /// <remarks>
-        /// This API creates a new individualEnrollment or update a existed one. All enrollments in the Device Provisioning Service
-        /// contains a unique identifier called registrationId. If this API is called for an individualEnrollment with a
-        /// registrationId that already exists, it will replace the existed individualEnrollment information by the new one.
-        /// On the other hand, if the registrationId does not exit, this API will create a new individualEnrollment.
-        ///
-        /// If the registrationId already exists, this method will update existed enrollments. Note that update the
-        /// individualEnrollment will not change the status of the device that was already registered using the old individualEnrollment.
-        ///
-        /// To use the Device Provisioning Service API, you must include the follow package on your application.
-        /// <c>
-        /// // Include the following using to use the Device Provisioning Service APIs.
-        /// using Microsoft.Azure.Devices.Provisioning.Service;
-        /// </c>
-        /// </remarks>
-        /// <example>
-        /// The follow code will create a new individualEnrollment that will provisioning the registrationid-1 using TPM attestation.
-        /// <c>
-        /// // IndividualEnrollment information.
-        /// private const string PROVISIONING_CONNECTION_STRING = "HostName=ContosoProvisioning.azure-devices-provisioning.net;" +
```

Comment:
Interesting that a connection string was hardcoded into the sdk before.

Reply:
This was a mock connection string thankfully. We obviously should not put live connection strings into the SDK's source code